src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -610,7 +611,7 @@ private function addTemplatingSection(ArrayNodeDefinition $rootNode)
                         ->then(function () { return ['enabled' => false, 'engines' => false]; })
                     ->end()
                     ->children()
-                        ->scalarNode('hinclude_default_template')->defaultNull()->end()
+                        ->scalarNode('hinclude_default_template')->setDeprecated('Using the key "hinclude_default_template" in the templating is deprecated since Symfony 4.3. Use the config key under fragments instead.')->defaultNull()->end()
```

Comment:
`Setting "templating.hinclude_default_template" is deprecated since Symfony 4.3, use "fragments.fragments.hinclude_default_template" instead.`